src/store/modules/governance/ethGovernance.ts

```diff
@@ -571,15 +571,22 @@ export class EthereumGovernance extends VuexModule.With({
 
     let metadata;
 
-    for await (const file of ipfs.get(hash, {
+    for await (const f of ipfs.get(hash, {
       timeout: timeoutInSeconds * 1000
     })) {
+      const file: {
```

Comment:
This is done because the docs (although slightly wrong with an extra <>) show [this](https://github.com/ipfs/js-ipfs/blob/master/docs/core-api/FILES.md#ipfsgetipfspath-options)